src/wire/ipv6routing.rs

```diff
@@ -0,0 +1,664 @@
+use core::fmt;
+use {Error, Result};
+
+pub use super::IpProtocol as Protocol;
+pub use super::Ipv6Address as Address;
+
+enum_with_unknown! {
+    /// IPv6 Extension Routing Header Routing Type
+    pub doc enum Type(u8) {
+        /// Source Route (DEPRECATED)
+        ///
+        /// See http://www.iana.org/go/rfc5095 for details.
```

Comment:
Use the ietf. The iana links just 302 to the ietf. https://tools.ietf.org/html/rfc5095

Reply:
fixed